sh/tmuxcomplete

```diff
@@ -27,7 +27,8 @@
 # Words including 2000 lines of history per pane
 #     sh tmuxcomplete -c '-S -2000'
 
-if ! tmux info &> /dev/null; then
+tmux info &> /dev/null
+if [ "$?" != "0" ]; then
```

Comment:
https://github.com/koalaman/shellcheck is awesome though I don't actually use it much.